config/environments/development.rb

```diff
@@ -131,8 +126,9 @@
 
   ###################################################################
   # Location of the public and private key for the git user on the system
-  GITOLITE_SETTINGS = { public_key: "/home/git/git.pub",
-                        private_key: "/home/git/.ssh/id_rsa" }
+  GITOLITE_SETTINGS = { public_key: "/home/vagrant/git.pub",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.